elements/lisk-cryptography/src/hash.ts

```diff
@@ -41,12 +41,13 @@ export const hash = (data: Buffer | string, format?: string): Buffer => {
 	}
 
 	throw new Error(
-		`Unsupported data:${data} and format:${format}. Currently only Buffers or hex and utf8 strings are supported.`,
+		`Unsupported data:${data} and format:${format ??
```

Comment:
For my knowledge what's the difference between: 

```
format ?? 'undefined' 
```

and 

```
format || 'undefined' 
```

Reply:
`??` will only check null or undefined, but || is falsy check